x/did/keeper/v1/keeper_did.go

```diff
@@ -0,0 +1,110 @@
+package keeper
+
+import (
+	"strconv"
+
+	types "github.com/cheqd/cheqd-node/x/did/types/v1"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// GetDidCount get the total number of did
+func (k Keeper) GetDidCount(ctx *sdk.Context) uint64 {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.DidCountKey))
+	byteKey := types.KeyPrefix(types.DidCountKey)
+	bz := store.Get(byteKey)
+
+	// Count doesn't exist: no element
+	if bz == nil {
+		return 0
+	}
+
+	// Parse bytes
+	count, err := strconv.ParseUint(string(bz), 10, 64)
+	if err != nil {
+		// Panic because the count should be always formattable to iint64
+		panic("cannot decode count")
+	}
```

Comment:
Another point to decide whether we should panic or set to `null` instead?

Reply:
This method is not used